// [[Reason: TODO Nested access to expando function.]] ////

//// [tests/cases/compiler/contextualReturnTypeOfIIFE2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -3,24 +3,4 @@
 //// [contextualReturnTypeOfIIFE2.d.ts]
 declare namespace app {
     function foo(): void;
 }
-
-/// [Errors] ////
-
-contextualReturnTypeOfIIFE2.ts(5,1): error TS9023: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
-
-
-==== contextualReturnTypeOfIIFE2.ts (1 errors) ====
-    declare namespace app {
-      function foo(): void;
-    }
-    
-    app.foo.bar = (function () {
-    ~~~~~~~~~~~
-!!! error TS9023: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
-      const someFun = (arg: number) => {};
-      return { someFun };
-    })();
-    
-    app.foo.bar.someFun(1);
-    
\ No newline at end of file
